src/SFA.DAS.Reservations.Data/SFA.DAS.Reservations.Data.csproj

```diff
@@ -4,8 +4,8 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Proxies" Version="2.1.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Proxies" Version="2.2.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.0" />
```

Comment:
framework upgrade - just highlighting in case unintentional

Reply:
thanks man, yeah, one of the packages is for api we use is dependant on 2.2.